scripts/versions/releasenotes

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+import github
+import argparse
+import yaml
+from urlparse import urlparse
```

Comment:
not sure if we want to be python2/python3 here but in python3 this is `from urllib.parse import urlparse`